pages/common/hledger-ui.md

```diff
@@ -0,0 +1,32 @@
+# hledger-ui
+
+> A robust, friendly plain text accounting app (the terminal UI).
+> More information: <https://hledger.org>.
+
+- Start in the main menu screen, reading from the default journal file:
+
+`hledger-ui`
+
+- Start with a different color theme:
+
+`hledger-ui --theme dark`
+
+- Start in the balance sheet accounts screen, showing hierarchy, to level 3:
+
+`hledger-ui --bs --tree --depth 3`
+
+- Start in this account's screen, showing cleared txns, and reload on change:
+
+`hledger-ui --register assets:bank:checking --cleared --watch`
+
+- Read two journal files, and show amounts as current value when known:
+
+`hledger-ui --file 2024.journal --file 2024-prices.journal --value now`
+
+- Show command-line help:
+
+`hledger-ui --help`
+
+- Show the manual in Info format if possible:
+
+`hledger-ui --info`
```

Comment:
Ok. But just FYI I showed --help first because it is simpler and always works, whereas --info is more involved and will not work on all systems. Ie I wanted to make sure user's first attempt to get help is successful.

Reply:
> Also "information" would be wrong here, because I'm talking about GNU's [info](https://en.wikipedia.org/wiki/Info_(Unix)) system. I capitalised it because people unfamiliar with info might not recognise the word, but lower case would be fine too.

Oh, thanks for the clarification will revert this change in my suggestion.